Source/WebCore/editing/CompositeEditCommand.h

```diff
@@ -71,6 +71,7 @@ class EditCommandComposition : public UndoStep {
     static Ref<EditCommandComposition> create(Document&, const VisibleSelection& startingSelection, const VisibleSelection& endingSelection, EditAction);
 
     void unapply() override;
+    void unapply(bool silent);
```

Comment:
This should be a named enum. Currently it's unclear what the parameter does at the two call sites.

Reply:
I'd define this enum class like `enum class DoNotAddToUndoManager : bool { No, Yes }`.